plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/jdbc/data/JDBCContentLOB.java

```diff
@@ -32,6 +37,8 @@
  */
 public abstract class JDBCContentLOB extends JDBCContentAbstract implements DBDContent {
 
+    private static final String ERROR_MESSAGE_PARAM_IS_SHOWN = "error-message-is-shown";

```

Comment:
As it common execution context, I assume this parameter should be called like "lob-reading-error-message-is-shown".
By the way, I'm not sure, but is it possible to store this property like a field? Maybe there is no need to store it in execution context.

Reply:
Do we really want to bloat lob with such a specific field? Execution context seems to be the most appropriate place to store such an information. Moreover, by doing so, the message will be shown only once per session rather than every time you open a lob.

Reply:
I think that maybe execution context is a too common place. But I just asked for a reason. I thought that JDBCContentLOB is created not every time you open a lob.

Reply:
I believe they have discussed it before prior to making this decision.

I agree that the parameter should be renamed.

Reply:
Yep-yep, just forgotten